addons/binding/org.openhab.binding.mysensors/.gitignore

```diff
@@ -0,0 +1 @@
+/*.cached
```

Comment:
Does something in your project create the *.cached files / directories? If not, please remove project specific gitignore file